docs/json_data/sysfs_attributes_drm.json

```diff
@@ -539,6 +539,138 @@
+            "attribute": "freq0/max_freq",
+            "scalar": 1e6,
+            "description": "Maximum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MAX_CONTROL"
+        },
+        "MIN_FREQ": {
+            "attribute": "freq0/min_freq",
+            "scalar": 1e6,
+            "description": "Minimum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MIN_CONTROL"
+        },
+        "RP0_FREQ": {
+            "attribute": "freq0/rp0_freq",
+            "scalar": 1e6,
+            "description": "Maximum non-overclocked GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_MAX_AVAIL"
+        },
+        "RPA_FREQ": {
+            "attribute": "freq0/rpa_freq",
+            "scalar": 1e6,
+            "description": "Actual GPU RPA frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "RPE_FREQ": {
+            "attribute": "freq0/rpe_freq",
+            "scalar": 1e6,
+            "description": "GPU RPE frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "RPN_FREQ": {
+            "attribute": "freq0/rpn_freq",
+            "scalar": 1e6,
+            "description": "Minimum requestable GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_MIN_AVAIL"
+        },
+        "IDLE_RESIDENCY": {
+            "attribute": "gtidle/idle_residency_ms",
+            "scalar": 1e-3,
+            "description": "Time spent in idle state",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "seconds",
+            "writeable": false,
+            "alias": ""
+        },
+        "HWMON::ENERGY2_INPUT": {
+            "attribute": "energy2_input",
```

Comment:
Would be nice if we can dynamically determine the `energy#_input` mappings based on the `name` attribute

Reply:
Ah, the hwmon docs you mention in the next file specify the # mapping. They state energy2 is package level and energy1 is card level, so we need to update this.